styles/style.css

```diff
@@ -1,26 +1,25 @@
 .advantages {
   text-align: center;
   background-color: #ffffff;
-  width: 1440px;
-  height: 570px;
-  margin-left: auto;
-  margin-right: auto;
+  max-width: 1440px;
+  max-height: 570px;
+  margin: auto;
```

Comment:
for margin-top/bottom better to add some value

Reply:
added value 0